addon/components/bsy-nav/item.js

```diff
@@ -6,14 +6,14 @@ export default Component.extend({
   layout,
   tagName: 'li',
   classNames: 'nav-item',
-  classNameBindings: ["activeClassName"],
+  classNameBindings: ['activeClassName'],
   active: false,
 
   activeClassName: computed('active', function() {
```

Comment:
Instead of this whole CP we can just put `active` into `classNameBindings` if I get it right.

Reply:
Same in `addon/components/bsy-nav/link.js`.

Reply:
Nice catch